src/drivers/imx/edma.c

```diff
@@ -25,6 +25,21 @@ DECLARE_SOF_UUID("edma", edma_uuid, 0x3d73a110, 0x0930, 0x457f,
 
 DECLARE_TR_CTX(edma_tr, SOF_UUID(edma_uuid), LOG_LEVEL_INFO);
```

Comment:
Please always explain why the commit is needed. 'Prepare for 8ulp' is not really self-explanatory. Also the code is not self explanatory.

When creating the commit please explain everything as you would want other to explain their code so that you can understand it.